src/main/java/ee/tuleva/onboarding/auth/idcard/IdCardSession.java

```diff
@@ -11,10 +12,15 @@
 @Builder
 public class IdCardSession implements Person, Serializable {
 
-  private static final long serialVersionUID = -111852724795571891L;
+  @Serial private static final long serialVersionUID = -111852724795571891L;
 
   public final String firstName;
   public final String lastName;
   public final String personalCode;
   public final IdDocumentType documentType;
+
+  @Override
+  public String getPhoneNumber() {
+    return null;
+  }
```

Comment:
We should not have the phoneNumber as part of the top-level interface if it's not relevant for _all_ the implementation classes.